### ISSUE TITLE
See number of workers

### ISSUE THREAD

**Issue by Loobinex** [OpenDungeons/OpenDungeons#1152]
It is currently not easy to know how much workers you have. Currently the only available options are to pick them all up or to look at how much mana the create worker spell will costs.

However, I've found myself building the maximum numbers of workers, only to find out much later that I had lost almost all of them in battle. I'd like to see a simple count of the amount of workers somewhere.

**Comment by hwoarangmy** [comment]
Indeed, the information is usefull but the GUI size is limited. I guess we could add that near the fighter number

**Comment by Loobinex** [comment]
Another possibility would be to put it as an overlay on the creature pane. You can do this both on the workers and the fighters.
For fighters right now it would be redundant information but my guess is that at some point this pane will show each creature type separately.

**Comment by Bertram25** [comment]
We have special buttons types permitting to show number and small icons next to the button image, so fortunately, this shouldn't be too hard to do. :)
